live.md

```diff
@@ -0,0 +1,60 @@
+
+## The topics
+
+The subject of these events have to be around the world of hosting and what the team usually works on. Among them we can have:
+
+* Server Environment (PHP, MySQL, nginx...)
+* Web Performance (caching, CDN, configuration...)
+* Reliability (backups, monitoring...)
+* Security (firewall, permissions, updates...)
+
+### Level of presentations
+
+The talks can be directed to users of any level. While it is true that local Meetups tend to reach all types of users, this channel was designed to raise the level of hosting professionals and teach or discuss topics that are not usually covered.
+
+Some ideas for presentation titles:
+
+* WordPress and the new PHP version
+* How we manage updates and maintenance
+* Optimizing hosting for WordPress sites
+* Complex and distributed WordPress installations
+
+## Languages
+
+The main vehicular language will be English, but it is not limited to English only. The main objective is to give a presentation in English at the established frequency, and in an interspersed manner, presentations in other languages can be included in the same format. The more people we can reach, the better!
+
+## Presentations
+
+As with any WordPress event, there are a number of limitations that apply across the board. These are included for the good of the global WordPress Community.
+
+* The first slide will include the title of the presentation, the name of the speaker and, if it is a derivative work, the original authors.
+* The second slide will include a presentation card of the speaker(s). In this slide you can inform about the company or business relationship you have, include logos, the page and social networks of the speaker.
+* The rest of the slides may not include any mention or commercial logos. As a general rule, no hashtags or social network accounts may be included.
+* The last slide can include a closing message (in Thank You! mode) as well as a link to the speaker's personal website, personal email account or personal social networks, in case anyone viewing the presentation wants to get in touch.
+* The presentation must be under GPL or compatible licenses (EUPL, etc...).
+* The [Code of Conduct](https://learn.wordpress.org/code-of-conduct/) must be observed at all times.
+
+## Propose a talk
+
+If you have an idea for a talk proposal, you can contact us proposing a topic from:
+
+* [WordPress Hosting Meetup](https://www.meetup.com/wphosting/discussions/)
+* [Make Hosting](https://make.wordpress.org/hosting/)
+* [#hosting-community](https://wordpress.slack.com/archives/hosting-community/) channel on Slack.
+
+\[info\]If you’re interested in improving this handbook, check the [Github Handbook repo](https://github.com/WordPress/hosting-handbook/), or leave a message in the [#hosting-community channel](https://wordpress.slack.com/archives/hosting-community/) of the official [WordPress Slack](https://make.wordpress.org/chat/).\[/info\]
```

Comment:
This is still the old version: https://github.com/WordPress/hosting-handbook/pull/81